src/engine/sync/synccontrol.cpp

```diff
@@ -516,7 +516,7 @@ void SyncControl::notifySeek(double dNewPlaypos) {
 double SyncControl::fileBpm() const {
     mixxx::BeatsPointer pBeats = m_pBeats;
     if (pBeats) {
-        return pBeats->getBpm();
+        return pBeats->getBpm().getValue();
     }
     return 0.0;
```

Comment:
return mixxx::Bpm::kValueUndefined instead of 0.0?

Reply:
https://github.com/mixxxdj/mixxx/pull/4045/commits/4cd4614c31fd7b1ddf230bb420395cf04f6c98c8